project/Build.scala

```diff
@@ -222,7 +222,9 @@ object Courier extends Build with OverridablePublishSettings {
     s";++$currentScalaVersion;project scala-generator;$publishCommand" +
     s";++$sbtScalaVersion;project sbt-plugin;$publishCommand" +
     s";++$sbtScalaVersion;project scala-runtime;$publishCommand" +
-    s";++$currentScalaVersion;project scala-runtime;$publishCommand"
+    s";++$currentScalaVersion;project scala-runtime;$publishCommand" +
+    s";++$sbtScalaVersion;project scala-fixture;$publishCommand" +
```

Comment:
Discussed offline. We'll keep artifacts for both scala versions, because they might be necessary and it doesn't hurt.